lecture/models.py

```
@@ -1,26 +1,55 @@
         super().save(*args, **kwargs)
+
+
+class LecturePDF(BaseModel):
+    lecture = models.ForeignKey(
+        Lecture,
+        on_delete=models.CASCADE,
+        related_name='pdf_files'
+    )
+    pdf_file = models.FileField(upload_to='lecture_pdfs/')
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)